spec/git_reflow/all-approval/git_reflow_spec.rb

```diff
@@ -18,6 +18,11 @@
   let(:existing_pull_request)  { GitReflow::GitServer::GitHub::PullRequest.new existing_pull_requests.first }
 
   before do
+
+    # Stubbing out numlgtm value to test all reviewers in gitconfig file
+    GitReflow::Config.stub(:get).with("constants.numlgtm").and_return('')
```

Comment:
We've been slowly converting our RSpec methods to use the new `allow` syntax instead of `stub`.  Would you mind updating these?  If it's too much effort don't worry about it and I'll create a task to do a global cleanup.